### DISCUSSION TITLE
Should board member be added to the organization's "people"?

### DISCUSSION THREAD

**Discussion by dodys** [OVAL-Community/OVAL#158] [General]
Hey,
Eduardo here from Canonical.
Just wondering if all board members should be added to the People section of this organization and if there should be any different levels of access.

**Comment by vanderpol** [comment]
Thanks for starting our first discussion, and I was just about to ask the same thing.  As NIWC is currently managing OVAL and the github repositories, I was planning to add all current OVAL board members as 'member' with the exception of my self, and a backup from NIWC.

It raises a second question, what do we do with people who are currently in the OVAL-Community 'people' that are not currently on the OVAL board?  I was thinking that converting them to "external contributor" made sense, but looking for feedback.  We have a couple people currently in the OVAL-Community that I cannot correlate to OVAL board members, but will be reaching out to determine who they are.

**Reply by vanderpol** [reply]
I also created an OVAL Board Members "Team", which may be useful if we want to get the attention of all OVAL board members in any discussions or votes.

**Comment by DeafBDoor** [comment]
I think for now that organization proposed by @vanderpol is good enough, including the use of external contributors.
Just to clarify: that means contributions done will only be merged by the two owners (or maintainers), right?
